spec/factories/users.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :user do
+    name {'taro'}
+    email {'abc@gmail.com'}
+    password {'password'}
+    password_confirmation {'password'}
+    admin {'false'}
```

Comment:
booleanの値をStringで設定するのは良くないです。
また、波括弧（ブレース）の中はスペース区切りにした方が見やすいです。
```suggestion
    admin { false }
```